windows_app_sdk/simple_window/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "simple_window"
+version = "0.0.0"
+edition = "2018"
+publish = false
+
+[dependencies]
+bindings = { package = "simple_window_bindings", path = "bindings" }
```

Comment:
Can we simplify this now that the workspace is gone?

Reply:
Good catch, will rename.